uitools/import/Esri/ArcGISRuntime/Toolkit/BasemapGallery.qml

```diff
@@ -241,6 +246,34 @@ Pane {
                 anchors.fill: parent
                 hoverEnabled: true
                 onClicked: controller.setCurrentBasemap(modelData.basemap)
+
+                // When mouse enters thumbnail area, use timer to delay showing of tooltip.
+                onEntered: {
+                    timerOnEntered.start();
+                }
+                // When mouse exits thumbnail area, use timer to delay hiding of tooltip.
+                onExited: {
+                    timerOnExited.start();
+                }
+
+                Timer {
```

Comment:
:thought_balloon: This is not a hard requirement. But with this implementation there are now 2 timers per basemapgallery item which is pretty costly. 

Since we only show 1 tooltip, there is possibly a saving to be made here by only having two timers for the entire gallery, if we keep them outside the gallery.

It would require passing a reference to the currently hovered basemap to the timer somehow.